corehq/apps/export/models/new.py

```diff
@@ -68,8 +72,9 @@ class ExportColumn(DocumentSchema):
     item = SchemaProperty(ExportItem)
     label = StringProperty()
     # Determines whether or not to show the column in the UI Config without clicking advanced
-    show = BooleanProperty(default=False)
+    is_advanced = BooleanProperty(default=False)
```

Comment:
I see, its a property that will only be shown when the user clicks a "show advanced" in the UI. Might be worth documenting that here on the model.